SCons/Node/FS.py

```diff
@@ -1199,9 +1201,31 @@ def __init__(self, path = None):
         DirNodeInfo.fs = self
         FileNodeInfo.fs = self
 
-    def set_SConstruct_dir(self, dir):
+    def set_SConstructs(self, dir, scripts=None):
+
```

Comment:
Add a docstring

Reply:
followed your advice from discord and just removed the unused functionality